src/Containers/CardCarousel/CardCarousel.tsx

```diff
@@ -0,0 +1,397 @@
+        33.3333333333% {
+            transform: translateY(-100%) scale(0.7);
+            opacity: 0.4;
+            visibility: visible;
+        }
+        36.3333333333% {
+            transform: translateY(-100%) scale(0.5);
+            opacity: 0;
+            visibility: visible;
+        }
+        100% {
+            transform: translateY(-100%) scale(0.5);
+            opacity: 0;
+            visibility: hidden;
+        }
+    }
+
+    @keyframes carousel-animate-vertical {
+        0% {
+            transform: translateY(-100%) scale(0.5);
+            opacity: 0;
+            visibility: hidden;
+        }
+        3%,
+        11.1111111111% {
+            transform: translateY(-100%) scale(0.7);
+            opacity: 0.4;
+            visibility: visible;
+        }
+        14.1111111111%,
+        22.2222222222% {
+            transform: translateY(0) scale(1);
+            opacity: 1;
+            visibility: visible;
+        }
+        25.2222222222%,
+        33.3333333333% {
+            transform: translateY(100%) scale(0.7);
+            opacity: 0.4;
+            visibility: visible;
+        }
+        36.3333333333% {
+            transform: translateY(100%) scale(0.5);
+            opacity: 0;
+            visibility: visible;
+        }
+        100% {
+            transform: translateY(100%) scale(0.5);
+            opacity: 0;
+            visibility: hidden;
+        }
+    }
+`;
+
+const CarouselCard = styled.div`
+    display: flex;
+    align-items: center;
+    position: absolute;
+    width: 500px;
+    height: 70px;
+    opacity: 0;
+    will-change: transform, opacity;
+    -webkit-animation: carousel-animate-vertical 27s infinite;
+    animation: carousel-animate-vertical 27s infinite;
+
+    @media (max-width: 600px) {
+        height: 65%;
+        width: 80%;
+    }
+
+    &:nth-child(1) {
+        -webkit-animation-delay: calc(3s * -1);
+        animation-delay: calc(3s * -1);
+    }
+
+    &:nth-child(2) {
+        -webkit-animation-delay: calc(3s * 0);
+        animation-delay: calc(3s * 0);
+    }
+
+    &:nth-child(3) {
+        -webkit-animation-delay: calc(3s * 1);
+        animation-delay: calc(3s * 1);
+    }
+
+    &:nth-child(4) {
+        -webkit-animation-delay: calc(3s * 2);
+        animation-delay: calc(3s * 2);
+    }
+
+    &:nth-child(5) {
+        -webkit-animation-delay: calc(3s * 3);
+        animation-delay: calc(3s * 3);
+    }
+
+    &:nth-child(6) {
+        -webkit-animation-delay: calc(3s * 4);
+        animation-delay: calc(3s * 4);
+    }
+
+    &:nth-child(7) {
+        -webkit-animation-delay: calc(3s * 5);
+        animation-delay: calc(3s * 5);
+    }
+
+    &:nth-child(8) {
+        -webkit-animation-delay: calc(3s * 6);
+        animation-delay: calc(3s * 6);
+    }
+
+    &:last-child {
+        -webkit-animation-delay: calc(-3s * 2);
+        animation-delay: calc(-3s * 2);
+    }
+`;
+
+const CardHead = styled.div`
+    background-color: #17be54;
```

Comment:
Try getting the closet color in the theme object or just use CheaprEats brand Colors